library/node

```diff
@@ -1,34 +1,43 @@
 # maintainer: Joyent Image Team <image-team@joyent.com> (@joyent)
 
-0.10.36: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
-0.10: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
-0: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
-latest: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
+0.10.36: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10
+0.10: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10
 
 0.10.36-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
 0.10-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
-0-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
 
-0.10.36-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
-0.10-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
-0-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
+0.10.36-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10/slim
+0.10-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10/slim
 
-0.11.16: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11
-0.11: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11
+0.11.16: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11
+0.11: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11
 
 0.11.16-onbuild: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/onbuild
 0.11-onbuild: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/onbuild
 
-0.11.16-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/slim
-0.11-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/slim
+0.11.16-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11/slim
+0.11-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11/slim
```

Comment:
Oh! OK, I did not know that was the case.

How about we leave this as is for now and I remove 0.11.6 in a follow up PR? I'd really like to get 0.12.0 push out...